src/ansible_navigator/ui_framework/menu_builder.py

```diff
@@ -81,14 +81,10 @@ def _menu_header_line(self, menu_layout: Tuple[List, ...]) -> CursesLine:
         """Generate the menu header line
 
         :param menu_layout: A tuple of menu details
-        :type menu_layout: tuple
-        :param menu_layout[0]: the starting in for each column
-        :type menu_layout[0]: List[int]
-        :param menu_layout[1]: the columns of the menu
-        :type menu_layout[1]: List[str]
-        :param menu_layout[2]: the adjusted column widths
-        :type menu_layout[2]: List[int]
-        :return: the menu head line
+            menu_layout[0]: List[int], the starting in for each column
```

Comment:
Is this rendered well?

Reply:
nope :)  made a list

Reply:
this one as well:
![image](https://user-images.githubusercontent.com/18386516/150585264-cff3bea8-2dad-46bd-89aa-5912af59e170.png)